notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>I think it is also important to note that NPU also support INT8 quantized models here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>